lib/ui/views/firstTime/theme.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:goipvc/ui/views/settings/theme.dart';
+
+class FirstTimeThemeSettingsView extends StatelessWidget {
+  const FirstTimeThemeSettingsView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Text(
+            "Customiza a tua app",
```

Comment:
Customiza doesn't sound right in my opinion
```suggestion
            "Personaliza a tua app",
```